pandas/_libs/tslibs/util.pxd

```diff
@@ -220,3 +223,8 @@ cdef inline const char* get_c_string_buf_and_size(str py_string,
 
 cdef inline const char* get_c_string(str py_string) except NULL:
     return get_c_string_buf_and_size(py_string, NULL)
+
+
+cdef inline object char_to_string_locale(const char* data):
```

Comment:
can the return here be more specific?

Reply:
I started with `str` but then realized that `char_to_string` has a return type of `object`, and the cython documentation says

> the generic [object](https://docs.python.org/3/library/functions.html#object) type (i.e. untyped code) will normally be good enough and has the additional advantage of supporting the assignment of string subtypes.

(see end of https://cython.readthedocs.io/en/latest/src/tutorial/strings.html)

So I just went for the same pattern as what was in place in pandas already for `char_to_string`

Reply:
makes sense, thanks